testbed/go.mod

```diff
@@ -6,12 +6,16 @@ require (
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
 	github.com/census-instrumentation/opencensus-proto v0.2.1
 	github.com/go-ole/go-ole v1.2.4 // indirect
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.3.4
 	github.com/open-telemetry/opentelemetry-collector v0.2.4-0.20200115225140-264426a9cae4
 	github.com/shirou/gopsutil v2.18.12+incompatible
 	github.com/spf13/viper v1.4.1-0.20190911140308-99520c81d86e
-	github.com/stretchr/testify v1.4.0
-	go.uber.org/zap v1.10.0
+	github.com/stretchr/testify v1.5.1
+	go.uber.org/zap v1.13.0
```

Comment:
Are these opportunistic updates or needed for this PR?

Reply:
needed for PR, a result of `go get -u` on the broken prometheus and k8s discovery, plus `go mod tidy`.

Reply:
I can try undoing those changes and re-running go mod tidy to see if they reappear, if you're extremely extremely concerned about staying on older versions if possible.

Reply:
I don't mind upgrading to newer versions of packages and go compiler but I'd prefer to do that in a separate PR if possible. If building on Windows _requires_ the newer go compiler and newer packages then I am fine with doing it in one PR.